index.html

```diff
@@ -8,7 +8,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700&display=swap" rel="stylesheet">
     <script src="https://kit.fontawesome.com/1bc3a7e061.js" crossorigin="anonymous"></script>
-    <title>Mobile first Portfolio</title>
+    <title>Belay Birhanu's Portfolio</title>
```

Comment:
As requested by the previous reviewers, the page's title is expected to adequately and briefly describe the content of the page. Kindly modify this and your project will be ready for approval.

Examples of poor titles:

- _Microverse project | Your portfolio._

- _Portifolio project_


Better page titles:

- _Mark's portifolio_

- _Mark full-stack webdeveloper_